app/data/test_language.json

```diff
@@ -11,18 +11,19 @@
         "id": "english-group",
         "title": "",
         "blocks": [{
-            "type": "Interstitial",
-            "id": "english-block",
-            "sections": [{
-                "title": "English language schema",
-                "id": "english-section",
-                "questions": []
-            }],
-            "title": ""
-        },
-        {
-            "type": "Summary",
-            "id": "summary"
-        }]
+                "type": "Interstitial",
+                "id": "english-block",
+                "sections": [{
+                    "title": "English language schema",
+                    "id": "english-section",
+                    "questions": []
+                }],
+                "title": ""
+            },
+            {
+                "type": "Summary",
+                "id": "summary"
+            }
+        ]
     }]
-}
+}
```

Comment:
I think they were all highlighted during a yarn lint, which was breaking my build locally. I think I made a separate commit for the JSON formatting, so feel free to revert that.